prow/def.bzl

```diff
@@ -48,6 +49,32 @@ def prow_image(
         **kwargs
     )
 
+# prow_image_arm64 is a macro for creating arm64 :app and :image targets
+def prow_image_arm64(
+        component,
+        name,  # use "image-arm64"
+        base = None,
+        stamp = True,  # stamp by default, but allow overrides
+        app_name = "app",
+        **kwargs):
+    go_image(
```

Comment:
Is there some important reason we need both `prow_image` and `prow_image_arm64` macros?

It would be nice to just add these two `go_image` and `container_image` calls to the `prow_image` macro, potentially gated by an `declare_arm = False` macro which we can set to true for the handful of places you want to use it

Reply:
My reasoning was that having arm images might not be required for everyone, and this way one can make them separately.
I'm not married to doing it this way, but I'd like to hear @stevekuznetsov's thoughts on this, too.

Reply:
Gating via some bool on the macro would achieve the same, though. I'll go ahead and make the change.